sparkle/src/components/Popover.tsx

```diff
@@ -0,0 +1,67 @@
+import * as PopoverPrimitive from "@radix-ui/react-popover";
+import * as React from "react";
+
+import { classNames } from "@sparkle/lib/utils";
```

Comment:
Let me merge https://github.com/dust-tt/dust/pull/7948 so you can use `cn` instead of `classNames`.

Reply:
Perfect

Reply:
Merged, but not published